templates/zerver/help/view-messages-containing-files-or-links.md

```diff
@@ -0,0 +1,10 @@
+# View messages containing files or links
+
+By using Zulip's [search feature](/help/search-for-messages), you can find all
+messages in your organization that contain any files or links.
```

Comment:
Is the "any" here needed? It seems kind of off. And maybe change "or" to "and/or".